### DISCUSSION TITLE
Cant get to configure waste_collection dialogue to complete

### DISCUSSION THREAD

**Discussion by Philg-home** [mampfes/hacs_waste_collection_schedule#3154] [General]
Hello, I am trying to get my waste collection setup but it always fails in the configuration dialogue.
I attach the waste fail.png for hopeful clarification.
I have tried one of the example test uprn but to no avail. Tried deleting waste schedule and re installing.
Welcome any suggestions.

![waste fail](https://github.com/user-attachments/assets/e567177e-7043-4217-be31-b4e4bf40e7ba)

**Comment by R4nd0m70** [comment]
Same for Carmarthenshire

**Comment by 5ila5** [comment]
I fixed the Wakefield and Carmarthenshire issue in #3191

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>

**Reply by Philg-home** [reply]
Thankyou for the update.
I followed the update instruction (easy) , got the message that i needed to reboot, so i did a full reboot. tried to setup but get the same message as per my diagram in first message.
I will wait for the full rollout to see if that resolves.
Many thanks. Phil

**Comment by Philg-home** [comment]
Cancel my last, I tried configure again 5 mins later and all appears to work, Thankyou very much.

**Comment by R4nd0m70** [comment]
Many thanks for updatingConfirmed working again 👍🏻 On 12 Dec 2024, at 12:05, 5ila5 ***@***.***> wrote:﻿
I fixed the Wakefield and Carmarthenshire issue in #3191
This will be part of the next release or is available now as master (git) version
install master/git: Developer tools -> actions -> select update: Install update -> click Coose entity and select WCS -> select version and type master in the input field -> run the action -> restart HA
Older versions of HACS:

install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
